keystone/gridder.py

```diff
@@ -206,23 +206,49 @@ def griddata(pixPerBeam=3.5,
     if outdir is None:
         outdir = os.getcwd()
     
+    if region=='NGC2264':
```

Comment:
with this section of the code growing, I would probably prefer to see this whole chunk here pulled out into a script maybe called `postprocess.py` which does clipping based on information in a version controlled text file (maybe in `yaml` format).  This would allow the code to be separate from the reduction parameters contained in the file.  If you think this is a good idea but aren't sure of how exactly to do this, we could work together to lay that out.  Thoughts?

Reply:
I think pulling this part out would be a good idea.  I'll create a yaml file and the postprocess.py script, along with a way to weave them into the griddata function